### ISSUE TITLE
File Manager Fails to unzip "large" files

### ISSUE THREAD

**Issue by Mohron** [Pterodactyl/Panel#185]
## Product
- [X] Panel

## Type
- [X] Bug or Issue

## What Happens
When attempting to unzip a 143MB zip file, the panel takes abnormally long and eventually fails. "Woops! an error while trying to process this request." Upon review of the files, some data was decompressed but the decompression never finishes.

## How to Reproduce
1. Upload a large zip file
2. Attempt to decompress using the file manager
## Error Logs

```
21:57:00.122Z ERROR wings:  (server=e525a791-6fb5-43c5-ae35-a8e844ecefc3, error={}, memo="self-thrown from progress event")
    file: {
      "name": "FTBInfinityLite110Server_1.2.0.zip",
      "mtime": "2016-11-28T21:53:27.617Z",
      "encoding": "octet",
      "clientDetail": {},
      "meta": {
        "path": "",
        "identifier": "5g44tx602cn88pzsjbovb73nmi"
      },
      "id": 7,
      "size": 150405576,
      "bytesLoaded": 100044800,
      "success": false,
      "base": "FTBInfinityLite110Server_1.2.0",
      "pathName": "/srv/daemon-data/infini_e525a791/data/FTBInfinityLite110Server_1.2.0.zip",
      "writeStream": {
        "_writableState": {
          "objectMode": false,
          "highWaterMark": 16384,
          "needDrain": false,
          "ending": false,
          "ended": false,
          "finished": false,
          "decodeStrings": true,
          "defaultEncoding": "utf8",
          "length": 0,
          "writing": false,
          "corked": 0,
          "sync": false,
          "bufferProcessing": false,
          "writecb": null,
          "writelen": 0,
          "bufferedRequest": null,
          "lastBufferedRequest": null,
          "pendingcb": 0,
          "prefinished": false,
          "errorEmitted": false,
          "bufferedRequestCount": 0,
          "corkedRequestsFree": {
            "next": null,
            "entry": null
          }
        },
        "writable": true,
        "domain": null,
        "_events": {},
        "_eventsCount": 3,
        "path": "/srv/daemon-data/infini_e525a791/data/FTBInfinityLite110Server_1.2.0.zip",
        "fd": 15,
        "flags": "w",
        "mode": "0666",
        "autoClose": true,
        "bytesWritten": 99942400
      }
    }
```

**Comment by DaneEveritt** [comment]
What happens if you set the `uploads.maxfilesize` to a higher value in the configuration?

**Comment by DaneEveritt** [comment]
I can confirm this is due to exceeding the maximum filesize limit set in the configuration.

**Comment by DaneEveritt** [comment]
closed in Pterodactyl/Daemon@f84877e and Pterodactyl/Panel@72ad6d5
